### DISCUSSION TITLE
I build a extension which can get CSS from figma, and built by wxt.

### DISCUSSION THREAD

**Discussion by zouhangwithsweet** [wxt-dev/wxt#545] [Show and tell]
wxt is awesome, it only took me a few minutes to read the documentation and develop using it.

Source code: [fubukicss-tool](https://github.com/zouhangwithsweet/fubukicss-tool)
Chrome store: [fubukicss-tool](https://chromewebstore.google.com/detail/fubukicss-tool/behnfolmiinfhphfdolomedncdnogcim)

**Comment by aklinker1** [comment]
That's cool! I've used figma for a while, but it's support for tailwind was never very good. I'll give your extension a go!
